index.html

```diff
@@ -84,10 +262,325 @@ <h4>Monday - Friday, July 19 - July 23, 2021</h4>
+                              <b>IET/Indiana Eastern Standard Time/GMT-5:00</b>
+                            </label>
+                          </div>
+
+                          <div class="form-check">
+                            <input class="form-check-input" type="radio" id="PRT" name="timezone" value="PRT" onclick="removeRemoteError()">
+                            <label class="form-check-label" for="PRT">
+                              <b>PRT/Puerto Rico and US Virgin Islands Time/GMT-4:00</b>
+                            </label>
+                          </div>
+
+                          <div class="form-check">
+                            <input class="form-check-input" type="radio" id="CNT" name="timezone" value="CNT"  onclick="removeRemoteError()">
+                            <label class="form-check-label" for="CNT">
+                              <b>CNT/Canada Newfoundland Time/GMT-3:30</b>
+                            </label>
+                          </div>
+
+                          <div class="form-check">
+                            <input class="form-check-input" type="radio" id="AGT" name="timezone" value="AGT"  onclick="removeRemoteError()">
+                            <label class="form-check-label" for="AGT">
+                              <b>AGT/Argentina Standard Time/GMT-3:00</b>
+                            </label>
+                          </div>
+
+                          <div class="form-check">
+                            <input class="form-check-input" type="radio" id="BET" name="timezone" value="BET"  onclick="removeRemoteError()">
+                            <label class="form-check-label" for="BET">
+                              <b>BET/Brazil Eastern Time/GMT-3:00</b>
+                            </label>
+                          </div>
+
+                          <div class="form-check">
+                            <input class="form-check-input" type="radio" id="CAT" name="timezone" value="CAT"  onclick="removeRemoteError()">
+                            <label class="form-check-label" for="CAT">
+                              <b>CAT/Central African Time/GMT-1:00</b>
+                            </label>
+                          </div>
+
+                          <div class="form-check">
+                            <input class="form-check-input" type="radio" id="UTC" name="timezone" value="UTC"  onclick="removeRemoteError()">
+                            <label class="form-check-label" for="UTC">
+                              <b>UTC/Universal Coordinated Time/GMT</b>
+                            </label>
+                          </div>
+
+                          <div class="form-check">
+                            <input class="form-check-input" type="radio" id="ECT" name="timezone" value="ECT" onclick="removeRemoteError()">
+                            <label class="form-check-label" for="ECT">
+                              <b>ECT/European Central Time/GMT+1:00</b>
```

Comment:
At the moment we are in summer time, so the relevant time timezone for people e.g. in France is CEST : GMT+2:00

Reply:
Changed!